domains/transfer-agreements/TransferAgreements.API/API/Metrics/TransferAgreementAutomationMetrics.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+
+namespace API.Metrics;
+
+public class TransferAgreementAutomationMetrics : ITransferAgreementAutomationMetrics
+{
+    public const string MetricName = "TransferAgreementAutomation";
+
+    private int numberOfTransferAgreementsOnLastRun = 0;
+    private ObservableGauge<int> NumberOfTransferAgreementsOnLastRun { get; }
+    private int certificatesTransferredOnLastRun = 0;
+    private ObservableGauge<int> CertificatesTransferredOnLastRun { get; }
```

Comment:
Why are these ObservableGauges and not Counters?

Reply:
Because they are totals and not counts